test/e2e/singlecluster/e2e_test.go

```diff
@@ -686,6 +687,57 @@ var _ = ginkgo.Describe("Kueue", func() {
 			}, util.LongTimeout, util.Interval).Should(gomega.BeTrue())
 		})
 
+		ginkgo.It("Should run with prebuilt workload", func() {
```

Comment:
Add ginkgo Steps to make the test easier to follow, below are just examples, step names as in integration tests are fine either.